docs/src/main/asciidoc/attributes.adoc

```diff
@@ -5,3 +5,11 @@
 :idseparator: -
 :icons: font
 :imagesdir: ./images
+
+//moved doctype to attributes.adoc to automatically enable it for all books when attribute file is included 
+:doctype: book
+
+:BrandName: Quarkus
```

Comment:
Another nit. Maybe go with lowercase instead of camelCase?

Reply:
I don't see why not. Thanks.